app/assets/stylesheets/blacklight_gallery.scss

```diff
@@ -1 +1,4 @@
-@import "blacklight_gallery/default";
+@import "blacklight_gallery/gallery";
```

Comment:
This _could_ be controversial? 🤷‍♂ but I think the intent is clear in not using the generated default anymore.